### DISCUSSION TITLE
Dropbox as a storage location

### DISCUSSION THREAD

**Discussion by jweston010** [AzuraCast/AzuraCast#3870] [Q&A]
i've got the latest rolling release, I've set up a remote dropbox storage location, after creating the developer app on Dropbox, but I'm not sure how to integrate for my existing station and set up. Do I delete the existing storage location, and then edit things so that my existing station uses Dropbox exclusively? As there's no step-by-step guide. Right now, my stations running on the local storage location.￼￼

**Comment by BusterNeece** [comment]
@jweston010 If you're moving all of your media over from where it is currently to Dropbox, then you can indeed just edit the existing storage location and change it to be a Dropbox one, and it'll start using Dropbox for your file management.

If you want to keep the old storage location for some stations and use Dropbox for others, you should create a new one instead.

Basically, every media storage location corresponds to every place you want to store media for at least one station.

**Comment by jweston010** [comment]
Now if I can just figure out my app problem.
There is a check box for app permission in dropbox, that is checked and 
grayed outt so I can uncheck it.
Azuracast says the permission is not set.
I think it is one of the read permissions so I can't even get the 
storage location set up anyway.
